src/components/notification/notificationShape.ts

```diff
@@ -0,0 +1,23 @@
+import { AssessmentCategory } from '../assessment/assessmentShape';
+
+export type AcademyNotification = {
+  assessment_id?: number;
+  assessment_type?: AssessmentCategory;
+  assesssment_title?: string;
+  id: number;
+  question_id?: number;
+  submission_id?: number;
+  type: AcademyNotificationType;
+};
+
+enum AcademyNotificationTypes {
```

Comment:
The backend passes the type as a string. In order to typecast from string to enum, we have to use `keyof typeof NotificationTypes`

Reply:
Yup. I meant why not export this and use it in other areas. e.g. instead of 

```
switch (type) {
    case 'new':
```

you could just do `AcademyNotificationTypes.new`